Gemfile.lock

```diff
@@ -42,29 +50,29 @@ GEM
       diff-lcs (>= 1.2.0, < 2.0)
       rspec-support (~> 3.9.0)
     rspec-support (3.9.3)
-    sys-uname (1.2.1)
-      ffi (>= 1.0.0)
-    typhoeus (1.4.0)
+    rubocop (0.66.0)
+      jaro_winkler (~> 1.5.1)
+      parallel (~> 1.10)
+      parser (>= 2.5, != 2.5.1.1)
+      psych (>= 3.1.0)
+      rainbow (>= 2.2.2, < 4.0)
+      ruby-progressbar (~> 1.7)
+      unicode-display_width (>= 1.4.0, < 1.6)
+    ruby-progressbar (1.13.0)
+    stringio (3.1.0)
+    typhoeus (1.4.1)
       ethon (>= 0.9.0)
-    vcr (3.0.3)
-    webmock (1.24.6)
-      addressable (>= 2.3.6)
-      crack (>= 0.3.2)
-      hashdiff
+    unicode-display_width (1.5.0)
 
 PLATFORMS
-  ruby
+  arm64-darwin-22
```

Comment:
@geoffharcourt I'm not sure if this might be an issue? I guess not a big issue, since CI is passing (even though it's linux, not darwin (MacOS))? But if I run `bundle install` on my machine, it adds `arm64-darwin-21`, which is a tiny bit annoying (and also sort of confusing?, because I am on Sonoma 14.1.1, which is `darwin-23` according to [this](https://en.wikipedia.org/wiki/Darwin_(operating_system)#Darwin_20_onwards)).

If we want to avoid these issues, I think we can do that by going back to the generic `ruby` platform, via something like `bundle lock --add-platform ruby && bundle lock --remove-platform arm64-darwin-22`.